packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Customize the Okta URL domain
+excerpt: Learn how to add a custom domain name to your Okta organization.
+layout: Guides
+---
+
+This guide teaches you how to customize the Okta URL domain which allows you to create a seamless branded experience for your users.
+
+---
+
+**Learning outcomes**
+
+Customize the Okta URL domain.
+
+**What you need**
+
+* A domain that you own, for example, `example.com`
+* A subdomain that you want to use, for example, `login.example.com`
+* A valid TLS certificate (PEM-encoded) for your subdomain
+* A 2048-bit private key (PEM-encoded)
+
+---
+
+## Overview
+
+You can customize your Okta organization by replacing the Okta domain name with your own domain name. This allows you to create a seamless branded experience for your users so that all URLs look like your application.
+
+Okta orgs host pages on subdomains such as `example.okta.com`. Using the custom URL domain feature aliases your Okta organization's domain name to another subdomain that you own, like `login.example.com`.
```

Comment:
This paragraph and the one below it seem repetitious — they are saying nearly the same thing. Could you combine them into one?